script.js

```diff
@@ -0,0 +1,68 @@
+const container= document.querySelector(".container");
+const count = document.querySelector("#count");
+const amount =document.getElementById("amount");
+const movie= document.getElementById("movie");
+const seats= document.querySelectorAll(".seat:not(.reserved)");
+getFromLocalStorage()
```

Comment:
Fonksiyon ismi daha aciklayici olmali. LocalStorage dan ne geliyor? getElementsFromLocalStorage mesela. Daha güzeli ve olmasi gereken elementi de aciklayarak yazmak. 
getSelectedSeatsFromLocalStorage. 
Aslinda senin fonksiyon sadece localStorage dan bir seyler almiyor. Baska isler de yapiyor. O yüzden isim daha da farkli olmali. Tam olarak yaptigi isi anlatan bir isim olmali.

Reply:
Kesinlikle katılıyorum sana benimde kafamı karaıştıran kısm bura oluyor zaten. 2 sebepten dolayı Hocaya sadık kalmayı tercih ediyorum. 1. meslenin mantığını kapmak daha öncelikli oduğu için durdurup anlamaya çalışıyorum. 2. değiştirdiğim zaman devamını takip etmek zor oluyor. bunlardan dolayı şimdiye kadar çok değişikliğe gitmedim ama bundan sonra daha açıklayıcı bir şekilde düzenleyerek gitmeyi deneyecem.

Reply:
İsimleri değiştirerek takip etmek de farkındalığı ve dikkati artırmaya zorluyor. Bazı derslerde yapmak olayı daha iyi anlamanı sağlar. Demeyebilirsin.